src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.container {
+  &:not(:nth-child(4n)) {
+    margin-right: 50px;
+  }
+  
+  &:nth-last-child(-n + 4) {
+    margin-top: 48px;
+  }
+
+  &__card {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.card {
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    position: relative;
+    left: 3px;
+    height: 134px;
+    width: 160px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    color: var(--color-black);
+  }
+  
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--color-grey);
+    margin-bottom: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: var(--color-grey);
+  }
+
+  &__price-cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: var(--color-black);
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 40px;
+    color: #fff;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    background-color: var(--color-blue);
+    text-transform: uppercase;
+    text-decoration: none;
+    border-radius: 5px;
+    border: 1px solid #ccc transparent;
+  }
+
+  &__link:hover {
+    background: #fff;
+    border: 1px solid var(--color-blue);
+    border-radius: 5px;
+    color: var(--color-blue);
+    height: 38px;
```

Comment:
why do you need this?

Reply:
is not fixed. 
Use padding instead of hardcode height